Documentation/tutorial/intro_toc.md

```diff
@@ -64,6 +64,7 @@ Property Name | Type              | Description
 ------------- | ----------------- | ---------------------------
 *name*        | string            | Specifies the title of the *TOC Item*.
 *href*        | string            | Specifies the hyperlink of the *TOC Item*.
+*tocHref*     | string            | Specified another TOC file to be expanded in the currrent *TOC Item*.
 *homepage*    | string            | Specifies the homepage of the *TOC Item*. It is useful when *href* is linking to a folder.
 *uid*         | string            | Specifies the `uid` of the referenced file. If the value is set, it overwrites the value of *href*.
 *homepageUid* | string            | Specifies the `uid` of the homepage. If the value is set, it overwrites the value of *homepage*.
```

Comment:
Yes, merged.

---
In reply to: [72408556](https://github.com/dotnet/docfx/pull/537#discussion_r72408556) [](ancestors = 72408556)